installer_scripts/install-yb-voyager-debezium

```diff
@@ -65,6 +65,48 @@ main() {
 # COMMON
 #=============================================================================
 
+# https://stackoverflow.com/a/4025065
+vercomp () {
+    if [[ $1 == $2 ]]
+    then
+        echo 0; return;
+    fi
+    local IFS=.
+    local i ver1=($1) ver2=($2)
+    # fill empty fields in ver1 with zeros
+    for ((i=${#ver1[@]}; i<${#ver2[@]}; i++))
+    do
+        ver1[i]=0
+    done
+    for ((i=0; i<${#ver1[@]}; i++))
+    do
+        if [[ -z ${ver2[i]} ]]
+        then
+            # fill empty fields in ver2 with zeros
+            ver2[i]=0
+        fi
+        if ((10#${ver1[i]} > 10#${ver2[i]}))
+        then
+            echo 1; return;
+        fi
+        if ((10#${ver1[i]} < 10#${ver2[i]}))
+        then
+            echo 2; return;
+        fi
+    done
+    echo 0; return;
+}
+
+# usage: version_satisfied <minimum_version_required> <actual_version>
+version_satisfied() {
```

Comment:
Hey @amit-yb do we need this abstraction in `version_satisfied` or can we just directly use the output of `vercomp`. We might encounter some cases in the future where we want the versions to be exactly equal and that cannot be checked using just the basic `-eq` for example `1.02` and `1.2`. 
In such a case we will anyway have to use `vercomp` directly.

Reply:
@ShivanshGahlot merging this for now. Don't see a strong reason for choosing either approach, so leaving it as is.